src/client/components/ReviewForm.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from "react";
+import postData from "./postData";
+
+function ReviewForm({ idOfMeal }) {
+  const date = new Date().toISOString().substring(0, 10);
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  const [rating, setRating] = useState("");
+
+  const newReview = {
+    title: title,
+    description: description,
+    meal_id: idOfMeal,
+    stars: rating,
+    created_date: date,
+  };
+  const handleAddReview = () => {
```

Comment:
Think about your error handling here. What happens when you throw the error? What will the user see?